### PR TITLE
Submit assignment

### PR DESCRIPTION
**Pull request by EriMiwa**
Derrick
I couldn't solve Q6's question of km/h. 
(expected data:0.4208, my answer:0.4209)

### CODE REVIEW

**Comment by itzderr** [comment]
I posted code for Question 6 on Slack #general

**Comment by itzderr** [comment]
Can you fix your code and send me a new pull request?

**Comment by EriMiwa** [comment]
Thanks for checking. all of answers fixed!

File: src/main/java/ca/ciccc/Assignment1.java [hunk 1]
```diff
@@ -127,7 +159,43 @@ public static String powers(int number) {
...
+        } else {
+            max = b;
+        }
+        int min;
+        if(a < b) {
+            min = a;
+        } else if( a == b) {
+            min = a;
+        } else {
+            min = b;
```
Comment (itzderr) [inline src/main/java/ca/ciccc/Assignment1.java]: You can also use `Math.min(a, b);` and `Math.max(a, b);`

**Comment by itzderr** [comment]
Nice

**Pull request closed without merging**